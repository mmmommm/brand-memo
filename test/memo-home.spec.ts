import MemoHome from '@/views/memo-home.vue'
import Vuetify from 'vuetify'

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe('memo-home.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  const wrapper = shallowMount(MemoHome, {
    localVue,
    vuetify,
  })
  it('snapshot test', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})