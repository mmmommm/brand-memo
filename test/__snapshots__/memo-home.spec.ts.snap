// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-home.vue snapshot test 1`] = `
<v-app-stub
  id="app"
>
  <loadingscreen-stub
    style="display: none;"
  />
   
  <layout-stub>
     
    <v-pagination-stub
      class="my-4"
      color="black"
      length="0"
      nexticon="$next"
      previcon="$prev"
      totalvisible="3"
      value="0"
    />
  </layout-stub>
</v-app-stub>
`;
